posydon/grids/termination_flags.py

```diff
@@ -99,20 +99,16 @@ def get_mass_transfer_flag(binary_history, history1, history2,
     Returns
     -------
     flag_system_evolution_history : string
-        Possible flags are: "contact during MS", "no_RLOF", a cumulative MT
-        flag + "_from_star1/2", or in case of missing binary history data,
-        "undetermined_flag_mass_transfer_with_no_binary_history".
+        Possible flags are: "None", "initial_RLOF", "contact_during_MS", 
+        "no_RLOF", a cumulative MT flag, e.g, "case_A1/B1/A2" where the
+        index indicates the donor star.
 
     """
-    if mesa_flag is not None:
-        if "overflow from L2" in mesa_flag:
-            if "at ZAMS" in mesa_flag:
-                return "initial_RLOF"
-            else:
-                return "L2_RLOF"
```

Comment:
This option `L2_RLOF` is not possible in the new version anymore. Is that correct, that it got removed?

Reply:
This option is obsolete, so I think it is okay to remove.